packages/language/src/v1beta1/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var languageServiceApi = require('./language_service_api');
+
+function v1beta1(options) {
```

Comment:
Would you mind doing a quick mock of what index.js would look like for PubSub? I think that would help me understand.